src/dmd/globals.d

```diff
@@ -148,7 +148,7 @@ extern (C++) struct Param
     bool betterC;           // be a "better C" compiler; no dependency on D runtime
     bool addMain;           // add a default main() function
     bool allInst;           // generate code for all template instantiations
-    bool fix16997;          // fix integral promotions for unary + - ~ operators
+    FeatureState fix16997;          // fix integral promotions for unary + - ~ operators
                             // https://issues.dlang.org/show_bug.cgi?id=16997
     bool fixAliasThis;      // if the current scope has an alias this, check it before searching upper scopes
```

Comment:
What subtype is Feature again? Byte I hope.

Reply:
[enum FeatureState : byte](https://github.com/dlang/dmd/blob/03d55018cff4bb3c4d75c7e4e356e393b8b37821/src/dmd/globals.d#L94)